tests/integration/scripts/setup.yml

```diff
@@ -13,6 +13,7 @@
         name: /girder_worker
         editable: true
 
+
```

Comment:
Any reason for this? Seems to be inconsistency in white space usage in this file.